test/utils/runSkuScriptInDir.js

```diff
@@ -1,14 +1,14 @@
 const spawn = require('./gracefulSpawn');
 const { exec } = require('child-process-promise');
 
-const scriptsPath = `${__dirname}/../../scripts/`;
+const skuBin = `${__dirname}/../../bin/sku.js`;
```

Comment:
Oh, awesome—I'd made the same change locally and was about to add it to your PR. Should have guessed you're way ahead of me 😄